packages/expo-ads-admob/ios/EXAdsAdMob/EXAdsAdMobRewarded.m

```diff
@@ -85,6 +86,16 @@ - (void)stopObserving {
       extras.additionalParameters = additionalRequestParams;
       [request registerAdNetworkExtras:extras];
     }
+    if(ssvOptions) {
+      GADServerSideVerificationOptions *options = [[GADServerSideVerificationOptions alloc] init];
+      if(ssvOptions[@"userId"]) {
+        options.userIdentifier = ssvOptions[@"userId"];
+      }
+      if(ssvOptions[@"customData"]) {
+        options.customRewardString = ssvOptions[@"customData"];
+      }
+      self.rewardedAd.serverSideVerificationOptions = options;
```

Comment:
same question as on the android side- what happens if one or both of `userId` or `customData` is missing? According to https://support.google.com/admob/answer/9603226?hl=en they are optional so I would assume nothing happens

Reply:
I've verified this too, it just performs all calls normally as if the GADServerSideVerificationOptions are not provided. (Server callback is called without user_id or custom_data in the querystring)

> Attached heroku log
<img width="716" alt="Screen Shot 2564-04-29 at 09 15 48" src="https://user-images.githubusercontent.com/5000572/116494656-7f26aa80-a8cb-11eb-9f22-1e3d8704166d.png">